README.md

```diff
@@ -49,6 +50,19 @@ or
 $ conda install -c omnia boost=1.58.0=py33_0
 ```
 
+Libgdf supports Apache Arrow versions 0.7.1 (default) and 0.8+ that
+use different metadata versions. So, it is important to specify which
+Apache arrow version will be used during building libgdf.  To select
+required Apache Arrow version, define the following environment
+variables (using Arrow version 0.9.0 as an example):
```

Comment:
@scopatz Actually, you are right. Otherwise, CI instances are using arrow 0.7.1. It might be reasonable to hold merging until pygdf is arrow 0.10.0 ready.

Reply:
I don't think it is worth holding off, we can always update this value for new arrow releases.

Reply:
It looks like in other places in the PR we're defaulting to 0.10.0 which I believe was released ~2 days ago. I'm in favor of having arrow 0.10.0 the default.